src/user/configure-programs-in-the-maintenance-app.md

```diff
@@ -440,7 +439,7 @@ number of days to wait for starting the program stage.
     for a repeatable stage after you click *Complete* for an event
     of the stage in data entry form. This field is selected by
     default.
-    5. Enter **Standard interval days**. The number of days to repeat the repeatable program stage.  
+    5. Enter **Standard interval days**. This value will be the suggested interval between the last event in a repeatable stage and the scheduled date of the next event.
     6. (Optional) Select a  **Default next scheduled date**. This will show a list of assigned data elements of type **date**. If an element is selected, the Tracker client will use this as the default starting date. The data element can be used by program rules to dynamically schedule intervals between events.
```

Comment:
I know this was not touched now - but baybe we could update this sentence a bit as well? 
>Tracker client will use this as the default starting date
...default scheduled date?

Reply:
done